cmd/sealer/cmd/run.go

```diff
@@ -52,6 +54,17 @@ create a cluster with custom environment variables:
 `,
 	Args: cobra.ExactArgs(1),
 	RunE: func(cmd *cobra.Command, args []string) error {
+		// set local ip address as master0 default ip if user input is empty.
+		// this is convenient to execute `sealer run` without set many arguments.
+		// Example looks like "sealer run kubernetes:v1.19.8"
```

Comment:
If we set this, we should clarify that very very clear in the command description or flag description.

Actually I like your design, but we should do more add complete the command and flag description. Otherwise it would only confuse to the end users.